packages/composables/src/useListing.ts

```diff
@@ -253,7 +254,7 @@ export function createListingComposable({
   //   ? `${contextName}(cms-${cmsContext})`
   //   : contextName;
 
-  const loading = ref(false);
+  const { loading } = useListingLoading();
```

Comment:
we can have multiple listings, for example listing and preview with another listing, cannot use the same loading state for both. `createListingComposable` is a factory so creates shared listing state composable for dedicated purpose. That would break some flows. What is the reason behind it?